nfsn-cron.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-set -o errexit
+set -eu
 
-cd $(dirname $0)
+cd "$(dirname "$0")"
 
```

Comment:
Given that this is NFSN, you would have to be braindead for this to be anywhere besides `/home/private/`, and if you deliberately change the names of your shell scripts to include spaces... well, that's gone right past braindead. Still, you can never be *too* careful, right? :-p